man/chage.1.xml

```diff
@@ -56,10 +56,12 @@
   <refsect1 id='description'>
     <title>DESCRIPTION</title>
     <para>
-      The <command>chage</command> command changes the number of days between
-      password changes and the date of the last password change. This
-      information is used by the system to determine when a user must change
-      their password.
+      The <command>chage</command> command changes password
+      expiration information for a user.
```

Comment:
"password expiration information" is a compound noun.  The three words should go in the same line.

I would break the line right after "changes".